docs/en/ingest-management/elastic-agent/install-on-kubernetes-using-helm.asciidoc

```diff
@@ -0,0 +1,24 @@
+[[install-on-kubernetes-using-helm]]
+= Install {agent} on Kubernetes using Helm
+
+preview::[]
+
+Starting with {stack} version 8.16, a Helm chart is available for installing {agent} in a Kubernetes environment. A Helm-based install offers several advantages, including simplified deployment, availability in marketplaces, streamlined ugrades, as well as quick rollbacks whenever they're needed.
+
+Features of the Helm-based {agent} install include:
+
+* Support for both standalone and {fleet}-managed {agent}.
+* For standalone agents, a built-in Kubernetes policy similar to that available in {fleet} for {fleet}-managed agents.
+* Support for custom integrations.
+* Support for {es} outputs with authentication through username and password, an API key, or a stored secret.
+* Easy switching between privileged (`root`) and unprivileged {agent} deployments.
+* Support for {stack} deployments on {eck}.
+
+For detailed install steps, try one of our walk-through examples:
+
+* <<example-kubernetes-standalone-agent-helm>>
+* <<example-kubernetes-fleet-managed-agent-helm>>
+
+NOTE: The {agent} Helm chart is currently available from inside the link:https://github.com/elastic/elastic-agent[elastic/elastic-agent] GitHub repo. It's planned to soon make the chart available from the Elastic Helm repository.
+
+You can also find details about the Helm chart, including all available YAML settings and descriptions, in the link:https://github.com/elastic/elastic-agent/tree/main/deploy/helm/elastic-agent[{agent} Helm Chart Readme].
```

Comment:
nit: maybe here it is worth mentioning also the examples in the repo if somebody wants to explore more use-cases?! https://github.com/elastic/elastic-agent/tree/main/deploy/helm/elastic-agent/examples 🙂

Reply:
For sure, let's include those. I've updated as shown:

![Screenshot 2024-10-31 at 11 28 14 AM](https://github.com/user-attachments/assets/7d7aef46-34b2-417f-b800-c35e6807b43c)